builder/builder-manifest.c

```diff
@@ -1883,7 +1883,8 @@ builder_manifest_create_platform (BuilderManifest *self,
             }
         }
 
-      if (self->separate_locales)
+      locale_dir = g_file_resolve_relative_path (root_dir, "share/runtime/locale");
```

Comment:
This will crash if separate_locales is false, because root_dir is not initialized then.